config/yaml_config.go

```diff
@@ -143,15 +143,15 @@ func mergeYamlConfig(agentConf *AgentConfig, yc *YamlAgentConfig) (*AgentConfig,
 	}
 	if yc.Process.Intervals.ContainerRealTime != 0 {
```

Comment:
The intervals are still using the old names, I'm looking for a way to support both the old and new without a lot of copy/paste